kinetic/distribution.yaml

```diff
@@ -2386,7 +2386,6 @@ repositories:
       - doosan_robotics
       - dsr_control
       - dsr_description
-      - dsr_example_cpp
```

Comment:
This is going to represent a regression for the users of the debian packages.  In particular, if they installed this package *before* this is synced, then they'll have access to the package, but nobody else will.  Is this what you mean to do?